island-is/island.is
# feat(samradsgatt): Add table of contents to the about page

# Add table of contents to about page

## What

Add table of contents to the about page

## Why

It is missing.

## Screenshots / Gifs

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/106164521/212652412-df45cfd9-311e-4cd4-b07d-142ba1e1ecbd.png">

Here is the figma desing: https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=209%3A8016&t=TMSB3vYqcaBWfSNr-0

## How to test

Checkout branch feature/samradsgatt/KAM-773-efnisyfirlit-a-about-sidu
Run samradsgatt locally: yarn start samradsgatt
Navigate to http://localhost:4200/samradsgatt/About
Look at the table of content at the top right of the page
Should look smiliar the figma design


## Checklist:
